internal/setup/setup.go

```diff
@@ -505,6 +512,9 @@ func parseRelease(baseDir, filePath string, data []byte) (*Release, error) {
 			}
 			archiveKeys = append(archiveKeys, key)
 		}
+		if details.Priority > MaxArchivePriority || details.Priority < MinArchivePriority {
+			return nil, fmt.Errorf("%s: archive %q has invalid priority value %v", fileName, archiveName, details.Priority)
```

Comment:
I think `%d` preferred over the general `%v`.

Reply:
Done in eef309d, thanks.